common-tasks/backup-restore.md

```diff
@@ -81,7 +135,11 @@ Emergency Backup Recovery without Qubes
 
 The Qubes backup system has been designed with emergency disaster recovery in mind. No special Qubes-specific tools are required to access data backed up by Qubes. In the event a Qubes system is unavailable, you can access your data on any GNU/Linux system with the following procedure.
 
-For emergency restore of backup created on Qubes R2 or newer take a look [here](/doc/backup-emergency-restore-v3/). For backups created on earlier Qubes version, take a look [here](/doc/backup-emergency-restore-v2/).
+Refer to the follow for emergency restore of a backup created on:
+
+ * [Qubes R4 or newer](/doc/backup-emergency-restore-v4/)
+ * [Qubes R3](/doc/backup-emergency-restore-v3/)
+ * [Earlier Qubes versions](/doc/backup-emergency-restore-v2/)
```

Comment:
Maybe "Qubes R2 or newer" for consistency?

Reply:
Or "older" I expect you meant! Updating.

Reply:
Yeah, that's what I meant.